stylus/settings/palette.styl

```diff
@@ -178,8 +178,8 @@ html, .CozyTheme--normal
 
     --primaryTextColor var(--charcoalGrey)
     --secondaryTextColor var(--coolGrey)
-    --primaryContrastTextColor var(--white)
-    --secondaryContrastTextColor var(--white)
+    --primaryContrastTextColor #FFF
+    --secondaryContrastTextColor #FFF
```

Comment:
Why do you use an absolute color instead of a CSS variables for white (and not for the other colors)?

Reply:
Because it makes no sense to have white as a variable (in the sense that it makes no sense to override the --white color in a theme). To me we should not use CSS variables here but stylus variables. CSS variables should be used only for themable colors (warnColor, validCOlor, primaryColor etc...).

I haven't changed everything here, it needs more work. For example we should in fact have two files, a default theme .styl and a palette.styl that defines all the shades of colors that we use, but as stylus variables.